jboss-kie-process-migration/module.yaml

```diff
@@ -4,3 +4,11 @@ version: '1.0'
 description: The jboss-kie-process-migration module.
 execute:
     - script: configure.sh
+artifacts:
```

Comment:
are these packages provided by Red Hat?

wouldn't be better installs it via microdnf/yum and them just do a symlink to the expected location?

Reply:
I think it has to be installed either by fetching the artifacts from maven repo or from the cekit cacher. Do you know a repo I could add to the content_sets.yaml?

Reply:
@ruromero , @spolti It looks like these are being installed as packages now... Is this good to go now?